doc/cargo_command_execution.md

```diff
@@ -75,6 +75,17 @@ The possible values:
 * Some object (`{ "RUST_BACKTRACE": 1 }`)
 * `null`
 
+### Setting An Action To Handle Starting A New Command If There Is Another Running Command
+
+The `"rust.actionOnStartingCommandIfThereIsRunningCommand"` configuration parameter specifies what the extension should do in case of starting a new command if there is another running command.
```

Comment:
`The `"rust.actionOnStartingCommandIfThereIsRunningCommand"` configuration parameter specifies what the extension should do in case of starting a new command if there is another command running.`